packages/jellyfish-api-core/__tests__/category/oracle/listPrices.test.ts

```diff
@@ -8,32 +8,23 @@ describe('Oracle', () => {
 
   beforeAll(async () => {
     await container.start()
-    await container.waitForReady()
     await container.waitForWalletCoinbaseMaturity()
   })
 
-  afterEach(async () => {
-    const data = await container.call('listoracles')
-
-    for (let i = 0; i < data.length; i += 1) {
-      await container.call('removeoracle', [data[i]])
-    }
-
-    await container.generate(1)
-  })
-
-  afterAll(async () => {
-    await container.stop()
-  })
-
-  it('should listPrices', async () => {
+  beforeAll(async () => {
```

Comment:
Maybe just a minor `nit`, but I think these tests are better for another PR, as they do not directly relate to consortium RPC.

Reply:
Hmm, yeah I've added it here as just the existing listPrices test was failing...

Reply:
Hmm, still think unrelated changes shouldn't be included, but I think these RPC are quite required rn, so let's get it in after its good to go.